### ISSUE TITLE
Windows could not start the NCR Counterpoint API service on Local Computer.

### ISSUE THREAD

**Issue by MikeRitchie** [NCRCounterpointAPI/APIGuide#1]
When manually starting the API service, the following error is displayed, and the service never successfully starts:

```
Windows could not start the NCR Counterpoint API service on Local Computer.

Error 1053: The service did not respond to the start or control request in a timely fashion.
```

The service automatically starts when the machine boots, but in that case the error above may not be displayed to the user, but it could still be found in the windows event log.

**Comment by MikeRitchie** [comment]
This will happen if the service can’t make all the DB connections, etc. in the default 30 second startup time. We are making changes before the API 1.0 release to speed up startup, but to workaround the issue you can increase the service startup timeout, it’s a registry key. 

http://serverfault.com/questions/622432/how-do-i-increase-windows-service-startup-timeout

I’d change to 60 seconds (60000) and try again. This could happen when:
•         The API server or DB server is really slow
•         The API is configured to connect to a lot of different companies/DBs
•         The API is configured to connect to a company where the DB/TLD isn’t there or slow to respond (the timeout makes the service wait).

Note that the setting is system wide, but it should have no negative impact on other services.

**Comment by mayurraichura** [comment]
Mike:

Is this API released in production?

**Comment by MikeRitchie** [comment]
Fixed in 1.0.212.
